pkgs/servers/matrix-synapse/default.nix

```diff
@@ -23,18 +23,22 @@ let
 
 in buildPythonApplication rec {
   pname = "matrix-synapse";
-  version = "0.99.5.2";
+  version = "1.0.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0c1kf9zq7cg9scwnvph4vwk449gypbprmdjzrzrg0wp1rcc8h3gn";
+    sha256 = "1n8hv0zd818z4fx39yz6svb07zsbrh8fd6wfmgvhdxhp6p1vl0wq";
   };
 
   patches = [
     # adds an entry point for the service
     ./homeserver-script.patch
   ];
 
+  postPatch = ''
+    substituteInPlace tests/http/__init__.py --replace '"openssl"' '"${openssl}/bin/openssl"'
+  '';
+
```

Comment:
Could we add openssl to `checkInputs` instead, perhaps?

Reply:
Ah, yes! Thanks!